functional/MockitoTests/playlist.xml

```diff
@@ -32,7 +32,7 @@
 			<group>functional</group>
 		</groups>
 		<versions>
-			<version>11+</version>
+			<version>[11,24]</version>
```

Comment:
Should we temporarily exclude this test via `disable`?

Reply:
Moved it to `<disables>`.